PIL/Image.py

```diff
@@ -2130,7 +2132,7 @@ def thumbnail(self, size, resample=BICUBIC):
     # FIXME: the different transform methods need further explanation
     # instead of bloating the method docs, add a separate chapter.
     def transform(self, size, method, data=None, resample=NEAREST, fill=1):
-        # type: (Size, int, Optional[Any], int, Any) -> Image
+        # type: (Size, int, Optional[Any], int, Any) -> Image  ## FIXME TYPING: What are Any here?
```

Comment:
data -> list[float]

Fill, I think is unused, but remains to not break things.